recipes-bsp/u-boot/u-boot-distro-boot/uEnv.txt.in

```diff
@@ -108,6 +112,116 @@ bootcmd_boot=if test ${bootscript_debug} != 1; then \
                  fi; \
              fi || true
 
-bootcmd_run=run board_fixups && run check_rollback_needed && run set_bootargs && run set_fdt_path && \
+prog_fuses=if test -n "${fuse_prog_list}"; then \
+               for fuse_val in "${fuse_prog_list}"; do \
+                   if test "${fuse_fail}" != "1"; then \
+                       if test "${fuse_dry_run}" = "1"; then \
+                           echo "DRY RUN: fuse prog ${fuse_bank} ${fuse_word} ${fuse_val}"; \
+                       else \
+                           fuse prog -y ${fuse_bank} ${fuse_word} ${fuse_val}; \
+                       fi; \
+                       if test $? != "0"; then \
+                           env set fuse_fail 1; \
+                       fi; \
```

Comment:
Sure I could do that, I just personally find this more readable.

Reply:
It's fine either way. I just asked because I commonly use a shorter notation for single statements protected by a "short" `if` but that's an absolutely subjective choice. Feel free to keep it the way you prefer.